lib/Form/FormsRepository.php

```diff
@@ -18,4 +18,14 @@ protected function getEntityClassName() {
     return FormEntity::class;
   }
 
+  public function findAllNotDeleted() {
+    return $this->entityManager
+      ->createQueryBuilder()
+      ->select('f')
+      ->from(FormEntity::class, 'f')
+      ->where('f.deletedAt IS NULL')
+      ->getQuery()
+      ->getResult();
```

Comment:
We could add an "orderBy" here to return the forms in some defined order ("updated_at" maybe? or "name"?).